tools/src/main/scala/caliban/tools/Formatter.scala

```diff
@@ -24,8 +24,6 @@ object Formatter {
       val scalafmt =
         Scalafmt
           .create(this.getClass.getClassLoader)
-          .withRespectVersion(false)
-          .withDefaultVersion("2.7.5")                           // For retro-compatibility
```

Comment:
@guizmaii was this important?

Reply:
It was important for backward compatibility only

Reply:
FTR it prevents any updates of scalafmt

```
[error] /caliban/tools/src/main/scala/caliban/tools/Formatter.scala:28:12: method withDefaultVersion in trait Scalafmt is deprecated
[error]           .withDefaultVersion("2.7.5")                           // For retro-compatibility
[error]            ^
[error] /caliban/tools/src/main/scala/caliban/tools/Formatter.scala:27:12: method withRespectVersion in trait Scalafmt is deprecated
[error]           .withRespectVersion(false)
[error]            ^
[error] two errors found
[error] (tools / Compile / compileIncremental) Compilation failed

```

Reply:
But it looks like it's failing if the user doesn't specify any scalafmt file. We need to support that case (maybe use a fake one where we just add version 2.7.5).

Reply:
> FTR it prevents any updates of scalafmt

I think it's more the dependency of Caliban on `scalafmt-dynamic` that prevents that.